tests/unit/query_graph/test_interpreter.py

```diff
@@ -312,20 +312,14 @@ def test_graph_interpreter_on_demand_tile_gen(
     expected_sql = textwrap.dedent(
         f"""
         SELECT
-          TO_TIMESTAMP(
-            DATE_PART(EPOCH_SECOND, CAST(__FB_START_DATE AS TIMESTAMPNTZ)) + tile_index * 3600
-          ) AS __FB_TILE_START_DATE_COLUMN,
```

Comment:
Previously, `F_TIMESTAMP_TO_INDEX` is called on this intermediate column later by tile manager. But the function works the same way if event timestamp column is passed in directly, so creating this intermediate column is not necessary.